.circleci/config.yml

```diff
@@ -1,12 +1,12 @@
 version: 2.1 
 
 orbs:
-  ruby: circleci/ruby@0.1.2 # Ruby orb registry: https://circleci.com/orbs/registry/orb/circleci/ruby
+  ruby: circleci/ruby@2.1.3 # Ruby orb registry: https://circleci.com/orbs/registry/orb/circleci/ruby
 
 jobs: 
   build: 
     docker: 
-      - image: circleci/ruby:2.4
+      - image: circleci/ruby:2.7
```

Comment:
This is to get rid of [these filter_map errors](https://app.circleci.com/pipelines/github/WeInfuse/change_health/313/workflows/21d98222-0bab-4f87-a65e-254ecfcea77c/jobs/243). Ruby 2.7+ has filter_map.